tests/test_app.py

```diff
@@ -35,7 +36,7 @@ async def test_startup_executors():
     def func(*args):
         return sum(args)
 
-    result = await app["run_in_thread"](func, 1, 5, 6, 2)
+    result = await virtool.utils.run_in_thread(func, 1, 5, 6, 2)
```

Comment:
This test doesn't make sense anymore as you remove `app["run_in_thread"]`.

Add a test for your new `run_in_thread` utility function that ensure the following:
* It is _awaitable_.
* It takes `args` and `kwargs` as expected and passes them to the `func`.
* The result of `func` is returned by `run_in_thread`.